.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,34 @@
 <!---
+
 Thanks for creating a pull request!
 
-Please start your PR title with the appropriate prefix as specified by [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#specification):
+## TODO
 
-chore: [your title]
-docs: [your title]
-feat: [your title]
-fix: [your title]
-refactor: [your title]
-rfc: [your title]
-test: [your title]
--->
+ - 🚧 Documentation in README, crate, module, function and/or other locations added/adapted.
 
+ - 🚧 New tests added and/or existing tests adapted.
 
-## Intent
-<!---
-Please describe what you want to achieve here.
-Keep your PR limited to one logical change.
-For example, if you add a feature, don't include fixes you made along the way
-Please open another PR for more contributions.
--->
+ - 🚧 PR is limited to a single logical change.
+   For example, if you add a feature, don't include fixes you made along the way
 
-## Description
-<!--- 
-For complex PRs, please describe what you did to achieve your goal here.
-Did you consider different approaches, where there any trade-offs you made?
-More information about why you implemented your solution the way you did helps reviewers.
-This section can be removed for obvious changes like typo fixes.
--->
+ - 🚧 At least the first commit has a clear title and a descriptive message containing reasoning for the change.
+   This will be included in the final commit message after squashing and merging.
+
+ - 🚧 PR title starts with an appropriate prefix as specified by [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#specification):
+
+       chore: [your title]
+       docs: [your title]
+       feat: [your title]
+       fix: [your title]
+       refactor: [your title]
+       test: [your title]
+
```

Comment:
I would keep whatever regarding the PR title, body, description, etc at the very top.

This way it will be the first thing contributors will read when creating a PR.

In short i prefer listing requirements:
1. About PR title, description, linked issues, etc.
2. About commit messages.
3. About the PRs content (the code).

Reply:
Mmmmm okay, I didn't think about it like this.

The reason why I prefer the other way around is because PR related requirements are be more accessible when opening the PR. We can do it in the same page and while creating it. Then, as we go deeper in our requirements, we can mark it as a draft and then handle anything else (commits, doc, tests, etc). So it feels more natural to me.

Reply:
(But I'm fine with any order if we get some more consensus on what makes sense to more people).